site/assets/scss/_styles_project.scss

```
@@ -475,6 +475,102 @@ h1.text-center {
+  text-align: left;
+
+  // Tab bar sits flush on top of the code block.
+  .nav-tabs .nav-item .nav-link.active {
+    background: #0d1117;
+    border-bottom-color: #0d1117;
+  }
+
+  .highlight {
+    margin: 0;
+    padding: 0;
+    // Grow the box to the width of the code (up to the viewport).
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)